recipes-security/optee-imx/optee-test_3.2.0.imx.bb

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2017-2018 NXP
+
+SUMMARY = "OPTEE test"
+HOMEPAGE = "http://www.optee.org/"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=daa2bcccc666345ab8940aab1315a4fa"
+
```

Comment:
`DEPENDS` comes before `inherit`.

Reply:
Forgot this one in the second revision, just pushed a third which includes this.